webapp/channels/src/components/delete_category_modal/delete_category_modal.tsx

```diff
@@ -48,9 +46,9 @@ export default function DeleteCategoryModal(props: Props) {
             confirmButtonClassName={'delete'}
         >
             <span className='delete-category__helpText'>
-                <FormattedMarkdownMessage
+                <FormattedMessage
                     id='delete_category_modal.helpText'
-                    defaultMessage="Channels in **{category_name}** will move back to the Channels and Direct messages categories. You're not removed from any channels."
+                    defaultMessage="Channels in <b>{category_name}</b> will move back to the Channels and Direct messages categories. You're not removed from any channels."
```

Comment:
@Aryakoste can you refer the example PR https://github.com/mattermost/mattermost/pull/28438 
We need to create pass the b variable as XML template